src/fragments/30/fragment30.h

```diff
@@ -0,0 +1,8 @@
+#include "global.h"
+
+extern void func_80029048(s32 arg0, u8* arg1, u8* arg2, s32 arg3, s32 arg4);
+extern u8 D_88000000;
+extern u8 fragment22_ROM_START[];
+extern u8 fragment23_ROM_START[];
```

Comment:
Remove the `extern` from the function declaration, you don't need it on functions, the other 3 can all be removed too once you use the macro.